packages-next/admin-ui-utils/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './dataGetter';
```

Comment:
Made this so there's a place to put utils that will likely be more unstable than the admin ui itself so this package can be major bumped without the admin ui being major bumped